src/App.js

```diff
@@ -9,32 +10,39 @@ import { BrowserRouter as Router } from "react-router-dom";
  * Footer: Global Footer
  * Navbar: Global navigation bar
  */
-import {
-  Footer,
-  Navbar,
-} from "./components/molecules";
-// Routes
+import { Footer, Navbar } from "./components/molecules";
+// Starts the page on top when reloaded or redirected
+import { ScrollToTop } from "./components/atoms";
+
```

Comment:
```suggestion
```

Reply:
We split imports per header and a new line.